Connectors/Tekla/Speckle.Connector.Tekla2024/HostApp/TeklaIdleManager.cs

```diff
@@ -0,0 +1,31 @@
+using Speckle.Connectors.DUI.Bridge;
+using Tekla.Structures.Model;
+
+namespace Speckle.Connector.Tekla2024.HostApp;
+
+public sealed class TeklaIdleManager : AppIdleManager
+{
+  private readonly IIdleCallManager _idleCallManager;
+  private readonly Events _events;
+
+  public TeklaIdleManager(IIdleCallManager idleCallManager) : base(idleCallManager)
+  {
+    _idleCallManager = idleCallManager;
+    _events = new Events();
```

Comment:
I saw this also in SendBinding. Can we inject it from the very beginning and pass here via constructor arguments?